### DISCUSSION TITLE
Some doubts about variants

### DISCUSSION THREAD

**Discussion by adrianmorse** [GitbookIO/community#783] [0. General bugs and questions] [answered]
Hello, 
We are evaluating Gitbook and just have the free version right now. We have a few questions about variants:
i. Suppose I have the original site and a variant. Is it possible to set things up such that only certain visitors can see the variant? Eg. I want employees to see all entries in the dropdown list, but I want "user group A" to just see a single entry.
ii. Can reusable content be reused across spaces/variants? E.g I create a reusable paragraph block in one space...can I use it in another or it is tied to that space?
Thanks
Adrian

**Comment by addisonschultz** [comment, marked as answer]
Hey Adrian

Great questions - right now this isn't possible to do exactly what you're looking for, as the visitor access we do have is tied to the space level - but it's an interesting use case and I think fits into some of the work we'd like to do in order to extend this product area. I'm going to pass this over to our product team to look into a bit more!

For your second point, unfortunately reusable content is tied only to the space it's in, but could be something we could explore in the future

**Reply by adrianmorse** [reply]
Thanks for the clarifications!
